Dockerfile

```diff
@@ -1,24 +1,30 @@
 FROM ubuntu:20.04
-
-# update system dependencies
 RUN apt update
 
-# install python depdendencies
-RUN apt install -y python3-dev python3-pip
-
-# create the app dir
-RUN mkdir -p /home/was/current
+# install python and ruby
+RUN apt install -y python3-dev python3-pip ruby ruby-dev
+RUN gem install bundler
 
 # create the directory where WARC data and CDX indexes live
-RUN mkdir -p /web-archiving-stacks/data/collections
-RUN mkdir /web-archiving-stacks/data/indexes
+RUN mkdir -p /web-archiving-stacks/data/collections/
+RUN mkdir /web-archiving-stacks/data/indexes/
```

Comment:
The `/web-archiving-stacks` directory doesn't exist in the new container, so that -p is needed for the first. But subsequently it's not needed since all we are doing is creating the indexes subdirectory in one that was just created?